### DISCUSSION TITLE
コメントへのコメントで、元コメントが削除されたときの対応

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#322] [要望・アイデア・不具合]
Ａさんの記事に対してＢさんがコメントをつけます。そのコメントに対してさらに第三者のＣさんがコメントしたとします。

その後、Ｂさんが自主的にコメントを削除すると、画面表示にはコメントＢの痕跡は全く残らず綺麗に消えてしまいます。そうなると、Ｃさんのコメントは宙に浮き、全く意味が通らない徘徊老人の独り言のような位置づけになってしまいます。

Qiitaの判断で削除したときには、「サービス利用規約に基づき、このコメントは削除されました。」という形で痕跡が残るようです。コメント主が自主的に削除したときにも、何等かの表示が残るようにしていただけないでしょうか？

これがあれば、徘徊老人のようには見られずに済みます。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
ありがとうございます🙏
コメント表示の見せ方は改善を予定しておりまして、その際にこちらも検討します！
